lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,878 @@
+            minor:
+              description: Extension minor version.
+              returned: always
+              type: int
+              sample: 0
+        nspname:
+          description: Namespace where the extension is.
+          returned: always
+          type: str
+          sample: pg_catalog
+    languages:
+      description: Procedural languages U(https://www.postgresql.org/docs/current/xplang.html).
+      returned: always
+      type: dict
+      sample: sql
+      contains:
+        lanacl:
+          description:
+          - Language access privileges
+            U(https://www.postgresql.org/docs/current/catalog-pg-language.html).
+          returned: always
+          type: str
+          sample: "{postgres=UC/postgres,=U/postgres}"
+        lanowner:
+          description:
+          - Language owner U(https://www.postgresql.org/docs/current/catalog-pg-language.html).
+          returned: always
+          type: str
+          sample: postgres
+    namespaces:
+      description:
+      - Namespaces (schema) U(https://www.postgresql.org/docs/current/sql-createschema.html).
+      returned: always
+      type: dict
+      sample: pg_catalog
+      contains:
+        nspacl:
+          description:
+          - Assess privileges U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+          returned: always
+          type: str
+          sample: "{postgres=UC/postgres,=U/postgres}"
+        nspowner:
+          description:
+          - Schema owner U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+          returned: always
+          type: str
+          sample: postgres
+    repl_slots:
+      description:
+      - Replication slots (available in 9.4 and later)
+        U(https://www.postgresql.org/docs/current/catalog-pg-replication-slots.html).
+      returned: if existent
+      type: dict
+      sample: slot0
+      contains:
+        active:
+          description: True if this slot is currently actively being used.
+          returned: always
+          type: bool
+          sample: true
+        database:
+          description: Database name this slot is associated with, or null.
+          returned: always
+          type: str
+          sample: acme
+        plugin:
+          description:
+          - The base name of the shared object containing the the output plugin
+            this logical slot is using, or null for physical slots.
+          returned: always
+          type: str
+          sample: pgoutput
+        slot_type:
+          description: The slot type - physical or logical.
+          returned: always
+          type: str
+          sample: logical
+    replications:
+      description:
+      - Information about the current replications by process PIDs
+        U(https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-STATS-VIEWS-TABLE).
+      returned: if pg_stat_replication view existent
+      type: dict
+      sample: 47823
```

Comment:
This seems weird.

Reply:
This is an example of a possible key of the dict. The key is a pid of a replication process that allows us to uniq it in a postgres instance. I'll try to change it to the dict

Reply:
Changed